eskip/template.go

```diff
@@ -37,17 +39,38 @@ func NewTemplate(template string) *Template {
 }
 
 // Apply evaluates the template using a TemplateGetter function to resolve the
-// placeholders.
-func (t *Template) Apply(get TemplateGetter) string {
+// placeholders. Returns true if all placeholders resolved to non-empty values.
+func (t *Template) Apply(get TemplateGetter) (string, bool) {
```

Comment:
Because we don't use `Apply()` anymore other than in `ApplyContext()`, I would rather not change the signature here (breaking change for library users).

Reply:
Ok, I've kept signature intact.